### DISCUSSION TITLE
Using raw subquery when defining filters in service/resolver

### DISCUSSION THREAD

**Discussion by mikitu** [doug-martin/nestjs-query#1233] [Q&A]
I need to filter something like this:
`const filters: Filter<Item> = {userId: {in:  Sequelize.literal(sql here)}}`
is it possible? 

Thanks

**Comment by awHamer** [comment]
No, the filters are high lever of abstraction that is not supposed to know something about SQL...
You can try to override`TypeormQueryBuilder` and pass it to constructor of `TypeormQueryService`, 
or you can add this directly to queryBuilder in `TypeormQueryService`.`query`
